definitions/hackingai.json

```
@@ -0,0 +1,32 @@
+{
+
+    "mlflow": {
+        "process_name": ["python*.exe",
+                         "python*",
+                         "pip*"],
+
+        "cmdline": ["mlflow"],
+        "domain": ["mlflow.org"]
+    },
+    "h2o.ai": {
+        "process_name": ["python*.exe",
+                         "python*",
+                         "pip*",
+                         "java*",
+                         "hadoop*"],
+        "cmdline": ["h2o-driver.jar", "h2odriver.jar", "h2o", "/h2o_"],
+        "domain": ["h2o.ai", "h2o-release.s3.amazonaws.com"],
+		"ipaddr":[
+			"228.246.114.236",
+			"ff05:0:3ff6:72ec:0:0:3ff6:72ec"
+		]
+    },
+    "ray.io": {
+        "process_name": ["python*.exe",
+                         "python*",
+                         "pip*",
+                         "gcs_server", "raylet"],
```

Remove generic process names to reduce false positives

```suggestion
        "process_name": ["raylet"],
```